pixi.toml

```diff
@@ -1,4 +1,4 @@
-[project]
+[workspace]
```

Comment:
Maybe we should add something like this https://github.com/holoviz/holoviews/pull/6699/files to the documentation.